extensions/label/README.md

```diff
@@ -59,10 +59,10 @@ Some additional notes are given here for some of the core STAC Item fields and w
 | label:overviews   | \[[Label Overview Object](#label-overview-object)] | Overview | An Object storing counts (for classification-type data) or summary statistics (for continuous numerical/regression data). |
 
 #### Class Object
-| Field Name | Type              | Name    | Description |
-| ---------- | ----------------- | ------- | ----------- |
-| name       | string\|null      | Name    | The property key within the asset's each `Feature` corresponding to class labels. If labels are raster-formatted, use null. |
-| classes    | \[string\|number] | Classes | The different possible class values within the property `name`. |
+| Field Name | Type                 | Name    | Description |
+| ---------- | -------------------- | ------- | ----------- |
+| name       | string\|null         | Name    | The property key within the asset's each `Feature` corresponding to class labels. If labels are raster-formatted, use null. |
```

Comment:
Wouldn't it make sense to require something here? Same for the other objects below...

Reply:
Yeah, agreed. We should make a separate issue for that.

Reply:
I see -- I misunderstood the relationship between your comments. I believe that 04edb4c addresses this as well.